src/svgcontext.ts

```diff
@@ -496,28 +500,29 @@ export class SVGContext implements RenderContext {
   }
 
   moveTo(x: number, y: number): this {
-    this.path += 'M' + x + ' ' + y;
+    this.path += 'M' + round3(x) + ' ' + round3(y);
     this.pen.x = x;
     this.pen.y = y;
     return this;
   }
 
   lineTo(x: number, y: number): this {
-    this.path += 'L' + x + ' ' + y;
+    this.path += 'L' + round3(x) + ' ' + round3(y);
     this.pen.x = x;
     this.pen.y = y;
     return this;
   }
 
   bezierCurveTo(x1: number, y1: number, x2: number, y2: number, x: number, y: number): this {
-    this.path += 'C' + x1 + ' ' + y1 + ',' + x2 + ' ' + y2 + ',' + x + ' ' + y;
+    this.path +=
+      'C' + round3(x1) + ' ' + round3(y1) + ',' + round3(x2) + ' ' + round3(y2) + ',' + round3(x) + ' ' + round3(y);
```

Comment:
maybe clearer as a backtick string?  With the r3 alias mentioned below?

Reply:
+1 for template strings